django/contrib/redirects/migrations/0001_initial.py

```diff
@@ -27,10 +20,15 @@ class Migration(migrations.Migration):
                     help_text="This can be either an absolute path (as above) or a full URL starting with 'http://'.",
                     max_length=200, verbose_name='redirect to', blank=True
                 )),
+                # We need to add `domain` field to migrate sites references
+                # data. Some db backends add fields by creating new table
+                # and copying all fields which are in project state. This
+                # field helps preserve `sites` references without requiring
+                # dependency on 'django.contrib.sites' migrations.
+                ('site_id', models.IntegerField(null=True)),
             ],
             options={
                 'ordering': ('old_path',),
-                'unique_together': {('site', 'old_path')},
```

Comment:
Same here, the unique constraint is only removed in the ORM but will still persist in the database.

Reply:
Since `site_id` is removed in further migration, it shouldn't be a problem.

Reply:
That implies that the constraint cascased, again I don't know if this is the case of all DBs.